docs/hstaxe/installing.rst

```diff
@@ -89,10 +78,13 @@ Python 3.9 due to missing ``tables`` wheels on PyPI).
 Legacy Astroconda Installation
 ------------------------------
 For historical preservation, we provide the original installation instructions
-for installing ``hstaxe`` via Astroconda:
+for installing ``hstaxe`` via Astroconda. We preserved a premade conda
+environment yaml in the repository for reproducability:
 
 .. code-block:: bash
 
+    conda create --name hstaxe-env --file legacy_astroconda_environment.yml
+    conda activate hstaxe-env
     conda install gsl cfitsio make automake autoconf libtool pkg-config -y
     conda install wcstools -c https://ssb.stsci.edu/astroconda -y
```

Comment:
Once the environment file is used, we do not need this... but we can keep this if we give both approaches separated by 'or'

Reply:
I am still a newbie with git reviews and comments:
By 'this' in the comment above, I meant: `conda install wcstools -c https://ssb.stsci.edu/astroconda -y`